app/views/admin_posts/index.html.erb

```diff
@@ -15,50 +15,30 @@
       </ul>
       <!--/subnav-->
     </div>
-  <!--main content-->
-  <% @admin_posts.each do |admin_post| %>
+    <!--main content-->
+    <% @admin_posts.each do |admin_post| %>
```

Comment:
Indenting and hash rocket updates from here on, with two exceptions I will point out...